### PR TITLE
Added valuesets for Service Request

### PR DESCRIPTION
**Pull request by Gokulsignal**
@enjoysparkling , Hi Matt, Kindly review the pull request.

### CODE REVIEW

File: input/vocabulary/valueset-service-request-order-detail.json
Comment (enjoysparkling) [inline input/vocabulary/valueset-service-request-order-detail.json:1]: Changes required:
```
Message  : Instance failed constraint vsd-10 "Must have a system if a code is present"
FilePath : input\vocabulary\valueset-service-request-order-detail.json
Id       : vsd-10
Severity : Error
Context  : ValueSet.expansion[0].contains[0]

```

File: input/vocabulary/valueset-us-core-service-request-category.json
Comment (enjoysparkling) [inline input/vocabulary/valueset-us-core-service-request-category.json:1]: Changes required:
- We should include the Service Type from HealthcareService in this field, see #72

**Review by enjoysparkling — CHANGES_REQUESTED** [review]
See notes in individual files.

File: input/vocabulary/valueset-us-core-service-request-category.json
Comment (arunsundar-m) [reply input/vocabulary/valueset-us-core-service-request-category.json:1]: @enjoysparkling Should we ignore this value set and bind the valueset-healthcare-service-type to this element in Forge?

File: input/vocabulary/valueset-service-request-order-detail.json
Comment (arunsundar-m) [reply input/vocabulary/valueset-service-request-order-detail.json:1]: @Gokulsignal Can you pick this change?

Comment (Gokulsignal) [reply input/vocabulary/valueset-service-request-order-detail.json:1]: The 'System' is added to the expansion.contains on 'valueset-service-request-order-detail'

File: input/vocabulary/valueset-us-core-service-request-category.json
Comment (enjoysparkling) [reply input/vocabulary/valueset-us-core-service-request-category.json:1]: We'll include this value set, but we will add to it in another task.

**Review by enjoysparkling — APPROVED** [review]
Looks good to go.

**Pull request merged**